ferrocene/doc/user-manual/src/rustfmt/cli.rst

```diff
@@ -8,23 +8,37 @@
 
    .. cli:option:: --check
 
-      Run in 'check' mode. Exits with 0 if input is formatted correctly. Exits with 1 and prints a diff if formatting is required.
+      Run in 'check' mode.
 
-   .. cli:option:: --emit [files|stdout|coverage|checkstyle|json]
+      Exits with status code 0 if input is formatted correctly.
+      Exits with status code 1 and prints a diff if formatting is not correct.
 
-      What data to emit and how
+   .. cli:option:: --emit [files|stdout]
+
+      What kind of output to emit.
+
+      - ``files``, which emits formatted output to files, is the default
+      - ``stdout`` emits the formatted output to stdout
 
    .. cli:option:: --backup
 
       Backup any modified files.
 
+      This copies the files before they are unmodified in same directory as the original files,
+      and changes the file extension to ``.bk``, replacing the ``.rs`` extension.
```

Comment:
Let's note that it overwrites existing `bk` files.

Reply:
85819601c15c532dc66cc9edb2943754660603de